tools/init.md

```diff
@@ -0,0 +1,45 @@
+# Starting a new project
+
+Starting a new project with Deno has always been incredibly simple: you just
+need a single file to get going. No need for any configuration files, dependency
+manifests, or build scripts.
+
+Users coming from other ecosystems are often not used to this simplicity - they
+often look for a tool to scaffold out a basic project structure to get them
+started on the right path. In this release we have added a deno init subcommand
+that scaffolds a basic Deno project.
```

Comment:
The phrase `In this release` doesn't make sense here. How about rephrasing this like:
```suggestion
started on the right path. `deno init` subcommand scaffolds a basic Deno
project.
```